code/modules/mob/mob.dm

```diff
@@ -4,12 +4,12 @@
 	GLOB.living_mob_list -= src
 	GLOB.mob_list -= src
 	unset_machine()
-	qdel(hud_used)
+	QDEL_NULL(hud_used)
+	QDEL_NULL(parallax)
```

Comment:
Wait a moment. Parallax is only created on client login, right? So if player's body is destroyed at the moment of their death, they will lose parallax until reconnect?

Reply:
Hmm. I have a local test up right now. The parallax does contain a reference to the parent mob, which is causing a hard del.

Reply:
I didn't lose the parallax in testing.